<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  
  
  <title>Data - AS3MT Paper</title>
  

  <link rel="shortcut icon" href="../img/favicon.ico">

  
  <link href='https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700' rel='stylesheet' type='text/css'>

  <link rel="stylesheet" href="../css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../css/theme_extra.css" type="text/css" />
  <link rel="stylesheet" href="../css/highlight.css">

  
  <script>
    // Current page data
    var mkdocs_page_name = "Data";
    var mkdocs_page_input_path = "data.md";
    var mkdocs_page_url = "/data/";
  </script>
  
  <script src="../js/jquery-2.1.1.min.js"></script>
  <script src="../js/modernizr-2.8.3.min.js"></script>
  <script type="text/javascript" src="../js/highlight.pack.js"></script>
  <script src="../js/theme.js"></script> 

  
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
      <div class="wy-side-nav-search">
        <a href=".." class="icon icon-home"> AS3MT Paper</a>
        <div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
        <ul class="current">
          
            <li>
    <li class="toctree-l1 ">
        <a class="" href="..">Home</a>
        
    </li>
<li>
          
            <li>
    <li class="toctree-l1 current">
        <a class="current" href="./">Data</a>
        
            <ul>
            
                <li class="toctree-l3"><a href="#processed-data">Processed Data</a></li>
                
            
                <li class="toctree-l3"><a href="#alignment-and-raw-data">Alignment and Raw Data</a></li>
                
            
            </ul>
        
    </li>
<li>
          
            <li>
    <li class="toctree-l1 ">
        <a class="" href="../contact/">Contact</a>
        
    </li>
<li>
          
        </ul>
      </div>
      &nbsp;
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="..">AS3MT Paper</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="..">Docs</a> &raquo;</li>
    
      
    
    <li>Data</li>
    <li class="wy-breadcrumbs-aside">
      
    </li>
  </ul>
  <hr/>
</div>
          <div role="main">
            <div class="section">
              
                <h2 id="processed-data">Processed Data</h2>
<p>The easiest way to access the underlying data from this paper is to use <a href="http://LieberInstitute.github.io/as3mt-paper/rdas/rawAndRpkmCounts_plusGenotype_10q24_DLPFC_n738.rda">this R object</a> which contains:</p>
<ol>
<li>Phenotype data: <code>pd</code>,  rows are subjects and columns are covariates and identifiers. Identifiers for the expression data are RNA Numbers (<code>Rxxxx</code>) and identifiers for the genotype data are Brain Numbers (<code>Brxxx</code>). Corresponding text file: <a href="http://LieberInstitute.github.io/as3mt-paper/data/phenotype_n738_LIBD.csv">pd</a></li>
<li>Gene-level data: <code>geneRpkm</code> and <code>geneCounts</code> correspond to gene RPKM and count values respectively and have genes/features down the rows and subjects/samples across the columns (in the same order as the rows of the phenotype data). <code>geneMap</code> is the corresponding annotation information. Corresponding text files: <a href="http://LieberInstitute.github.io/as3mt-paper/data/geneRpkm_n738_LIBD.csv">geneRpkm</a>, <a href="http://LieberInstitute.github.io/as3mt-paper/data/geneAnnotation_Ensembl75.csv">geneMap</a></li>
<li>Exon-level data: <code>exonRpkm</code> and <code>exonCounts</code> correspond to exon RPKM and count values respectively and have exons/features down the rows and subjects/samples across the columns (in the same order as the rows of the phenotype data). <code>exonMap</code> is the corresponding annotation information. Corresponding text files: <a href="http://LieberInstitute.github.io/as3mt-paper/data/exonRpkm_n738_LIBD.csv">exonRpkm</a>, <a href="http://LieberInstitute.github.io/as3mt-paper/data/exonAnnotation_Ensembl75.csv">exonMap</a></li>
<li>Junction-level data: <code>jRpkm</code> and <code>jCounts</code> correspond to junction RPMs and count values respectively and have exons/features down the rows and subjects/samples across the columns (in the same order as the rows of the phenotype data). <code>jMap</code> is the corresponding annotation information, which is a <code>GRanges</code> object. Corresponding text files: <a href="http://LieberInstitute.github.io/as3mt-paper/data/junctionRpm_n738_LIBD.csv">jRpm</a>, <a href="http://LieberInstitute.github.io/as3mt-paper/data/junctionAnnotation_LIBD.csv">jMap</a></li>
<li>Genotype data: <code>snpMat</code> contains the number of copies of the minor allele, and have variants down the rows and subjects/samples across the columns (in the same order as the rows of the phenotype data). <code>snpInfo</code> contains the corresponding annotation and GWAS statistics information. Corresponding text files: <a href="http://LieberInstitute.github.io/as3mt-paper/data/snpMinorCounts_LIBD.csv">snpMap</a>, <a href="http://LieberInstitute.github.io/as3mt-paper/data/snp_annotation.csv">snpInfo</a></li>
</ol>
<h2 id="alignment-and-raw-data">Alignment and Raw Data</h2>
<p>The phenotype object has public file paths for the 2 megabase region BAM files in the 10q24.32 genomic locus, and raw gene and exon counts files, which are all hosted on AWS S3 storage.</p>
<hr />
<p><a href="http://libd.org"> <img src="http://research.libd.org/as3mt-paper/images/LIBD_logo.jpg" width="200" align="middle"/></p>
              
            </div>
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="../contact/" class="btn btn-neutral float-right" title="Contact">Next <span class="icon icon-circle-arrow-right"></span></a>
      
      
        <a href=".." class="btn btn-neutral" title="Home"><span class="icon icon-circle-arrow-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="http://www.mkdocs.org">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
	  
        </div>
      </div>

    </section>

  </div>

<div class="rst-versions" role="note" style="cursor: pointer">
    <span class="rst-current-version" data-toggle="rst-current-version">
      
      
        <span><a href=".." style="color: #fcfcfc;">&laquo; Previous</a></span>
      
      
        <span style="margin-left: 15px"><a href="../contact/" style="color: #fcfcfc">Next &raquo;</a></span>
      
    </span>
</div>

</body>
</html>
